tests/informatics/test_design_spaces.py

```diff
@@ -124,14 +123,17 @@ def test_hierarchical_initialization(hierarchical_design_space):
 def test_data_source_build(valid_data_source_design_space_dict):
     ds = DesignSpace.build(valid_data_source_design_space_dict)
     assert ds.name == valid_data_source_design_space_dict["data"]["instance"]["name"]
+    assert ds.description == valid_data_source_design_space_dict["data"]["description"]
     assert ds.data_source == DataSource.build(valid_data_source_design_space_dict["data"]["instance"]["data_source"])
     assert str(ds) == f'<DataSourceDesignSpace \'{ds.name}\'>'
```

Comment:
```suggestion
    assert str(ds) == f"<DataSourceDesignSpace '{ds.name}'>"
```
As long as we're here.

Reply:
Sure.